packages/openmrs/src/http.js

```diff
@@ -42,3 +42,91 @@ export function request(method, path, options = {}, callback = s => s) {
     return util.prepareNextState(state, response, callback);
   };
 }
+
+/**
+ * Make a GET request to any OpenMRS endpoint
+ * @public
+ * @function
+ * @example
+ * get(
+ *  "/ws/rest/v1/patient",
+ *  {
+ *    query: {
+ *      limit: 1
+ *    }
+ *  }
+ *  )
+ * @param {string} path - path to resource
+ * @param {OpenMRSOptions} [options={}] - An object containing query params, headers and body for the request
+ * @returns {operation}
+ */
+export function get(path, options = {}, callback = s => s) {
+  return async state => {
+    const [resolvedPath, resolvedOptions] = expandReferences(state, path, options);
+
+    const response = await util.request(state, 'GET', resolvedPath, resolvedOptions);
+
+    return util.prepareNextState(state, response, callback);
+  };
+}
+
+
+/**
+ * Make a POST  request to an OpenMRS endpoint
+ * @public
+ * @function
+ * @example
+ * post(
+ *  "/ws/rest/v1/patient",
+ *  {
+ *    data: {
+ *      "person": {
+ *      "gender":"M",
+ *      "age":47,
+ *      "birthdate":"1970-01-01T00:00:00.000+0100",
+ *      "names":[
+ *        {
+ *          "givenName":"Jon",
+ *          "familyName":"Snow"
+ *        }
+ *      ],
+ *    }
+ *    }
+ *  }
+ * )
+ * @param {string} path - path to resource
+ * @param {OpenMRSOptions} [options={}] - An object containing query params, headers and body for the request
+ * @returns {operation}
+ */
+export function post(path, options = {}, callback = s => s) {
+  return  async state => {
+    const [resolvedPath, resolvedOptions] = expandReferences(state, path, options);
+
+    const response = await util.request(state, 'POST', resolvedPath, resolvedOptions);
+
+    return util.prepareNextState(state, response, callback);
+  }
+}
+
+
+/**
+ * Make a DELETE request to an OpenMRS endpoint
+ * @public
+ * @function
+ * @example
+ * remove(
+ *  "/ws/rest/v1/patient/abc/"
+ * )
+ * @param {string} path - path to resource
+ * @param {OpenMRSOptions} [options={}] - An object containing query params, headers and body for the request
+ * @returns {operation}
+ */
+export function remove(path, options = {}, callback = s => s) {
```

Comment:
This needs to be named `delete` rather than `remove`.

To avoid the js error, if you call it `_delete` I think you can do `export { _delete: delete` to alias it.

To fix the name in the generated docs (inside the openmrs folder, run `pnpm build` and look at the `docs` folder for the markdown), you can add the `@alias dete` tag https://jsdoc.app/tags-alias  That should work